### ISSUE TITLE
no PKZIPped HTML download from lynx

### ISSUE THREAD

**Issue by mirabilos** [FicHub/fichub.net#12]
I just requested an FFN story from lynx and was only offered EPUB and MOBI formats, no PKZIPped HTML. I had to fall back to Firefox (accessibility-wise problematic) in order to get PKZIPped HTML.

I just recommended FicHub to a blind lynx user who could _not_ do that because it worked with another FFN story I tested earlier, but this would be a problem for her.

(Also, what you write on the website, when you say…

> We currently recommend the EPUB format, as many EPUB readers have options to change font, font size, colors, and other options to enhance readability.

… is ridiculous, as I enjoy **much** more freedom of control of formatting with lynx: it runs in a terminal, with the color scheme (lynx style sheet) I set, the terminal font I use, etc. and it can be read using a Braille line, too, so the HTML format is definitively _much_ more versatile, and, in addition, a widely accepted standard. In contrast, EPUB packages some HTML with a homegrown index (though I’ve hacked together a shellscript to unpack EPUB and generate an `index.htm` from its `toc.ncx`(IIRC) file, that’s not as reliable), and I’ve never even seen what MOBI is supposedly good for.)

Long story short, plain HTML is a required first-class format.

It also doesn’t show up in the cache, for what it’s worth.

**Comment by iridescent-beacon** [comment]
It's great to hear that workflow is actually being used :)  I've made a change so generate links are presented when the secondary format is not already generated (or is out of date). Please let me know if there are any issues with it.

> Also, what you write on the website … _snip_ … is ridiculous

I'm happy I've attracted such passionate users! It's clear that you're an advanced user -- I don't think the average person knows enough to customize html rendering to any degree, and the ones who do should be able to figure out that they can simply download the zipped html if they wish to do that. I believe it makes the most sense to leave the wording as-is to help the most users.

Please feel free to open additional issues if you have other suggestions.

**Comment by mirabilos** [comment]
Hi iris,

>It's great to hear that workflow is actually being used :)  I've made a
>change so generate links are presented when the secondary format is not
>already generated (or is out of date). Please let me know if there are
>any issues with it.

thank you!

>I'm happy I've attracted such passionate users!

Well, it’s more like CloudFlare (or more specifically hCAPTCHA, which
is the service they use) insists on ignoring nōn-JS users pretending
using screenreaders with Internet Explorer is sufficient for accessi‐
bility… I’m primarily a lynx user.

>It's clear that you're an advanced user -- I don't think the average
>person knows enough to customize html rendering to any degree, and the

In lynx, it’s easy enough: the font is determined by the terminal it’s
run in. The .lss files are only if you want to customise the colours
assigned to tags or CSS classes, the defaults are quite good.

>Please feel free to open additional issues if you have other suggestions.

Thanks, will do if necessary. Kudos for lynx support in the first place
(from seeing the page in Firefox I had not expected it), and for the
fast help!

bye,
//mirabilos
-- 
  "Using Lynx is like wearing a really good pair of shades: cuts out
   the glare and harmful UV (ultra-vanity), and you feel so-o-o COOL."
                                         -- Henry Nelson, March 1999
